axum/src/extract/raw_form.rs

```diff
@@ -0,0 +1,54 @@
+use async_trait::async_trait;
+use axum_core::extract::FromRequest;
+use http::{Method, Request};
+
+use super::{
+    has_content_type,
+    rejection::{InvalidFormContentType, RawFormRejection},
+};
+
+use crate::{body::HttpBody, BoxError};
+
+/// Extractor that extracts the raw form string, without parsing it.
```

Comment:
Wanna mention that this looks at the query for GET and body for other methods, and that it requires the right content-type?